frontend/plugins/frontline_ui/src/modules/integrations/erxes-messenger/components/EMPreviewIntro.tsx

```
@@ -237,3 +576,298 @@ export const EMPreviewAuthForm = () => {
+                className={cn(
+                  { '-mt-8': index === 0 },
+                  'group rounded-2xl shadow-xs p-3.5 bg-background flex items-center gap-3 transition-all duration-300 ease-in-out hover:-translate-y-0.5 cursor-pointer hover:shadow-sm',
+                )}
+                role="button"
+                tabIndex={0}
+              >
+                <span
+                  className="shrink-0 p-2 rounded-xl flex items-center justify-center"
+                  style={{
+                    backgroundColor:
+                      `color-mix(in srgb, ${color} 60%, transparent)` ||
```

ESLint

[error] Unexpected constant truthiness on the left-hand side of a `||` expression.

(no-constant-binary-expression)